### ISSUE TITLE
Kubeflow installation issue(on prem)

### ISSUE THREAD

**Issue by Abhi3linku** [kubeflow/manifests#2244]
I am getting the below error while trying to deploy/install kubeflow in on-prem env. Few days back the installation was all working but with the new manifests now we are getting the error.

Env Details: Kubernetes v 1.21.7
Kustomize : 3.2.1
Deployment command: while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
Manifests: https://github.com/kubeflow/manifests#installation
Error:
unable to decode "STDIN": resource.metadataOnlyObject.ObjectMeta: v1.ObjectMeta.Annotations: ReadString: expects " or n, but found 4, error found in #10 byte of ...|hecksum":4.5463e+49}|..., bigger context ...|":{"annotations":{"knative.dev/example-checksum":4.5463e+49},"labels":{"app.kubernetes.io/component"|...

**Comment by kairos03** [comment]
Did you try checkout target version tag and apply on it?

**Comment by Abhi3linku** [comment]
@kairos03 Hi, I just git clone from the below location and ran the command from the manifests folder. Do you want to me try with any specific version?
https://github.com/kubeflow/manifests#installation

**Comment by kairos03** [comment]
@Abhi3linku Exactly, right.

**Comment by Abhi3linku** [comment]
I can see the issue is consistent in 1.5.0 and 1.5.1.

**Comment by kairos03** [comment]
Humm. it still? If then, I hardly suggest [install-individual-components](https://github.com/kubeflow/manifests#install-individual-components) :)

**Comment by Abhi3linku** [comment]
@kairos03  Sure Will try with individual components. However I have the manifests which worked before a month.

**Comment by kairos03** [comment]
You also can apply the manifests individually

**Comment by Abhi3linku** [comment]
@kairos03  Yes. As of now with my old manifests it works. Will apply the individual components if required. Thank You!

**Comment by kairos03** [comment]
@Abhi3linku That's sounds good :) Most of my caese "install with a single command" not work properly. I don't know why. So, I always install individually.

**Comment by hoangtnm** [comment]
Hi @kairos03, I'm new to kubeflow manifests, and I'm not so sure how to install Kubernetes so that it can work well with kubeflow manifests. Do you have any docs for steps that you ran on your system? Thank you

**Comment by juliusvonkohout** [comment]
/close

There has been no activity for a long time. Please reopen if necessary.
